internal/install/recipe_installer_test.go

```diff
@@ -132,75 +134,12 @@ func TestInstall_FailsOnInvalidOs(t *testing.T) {
 		},
 	}
 
-	i := RecipeInstaller{ic, discover, l, f, e, v, ff, status, p, pi, lkf}
+	i := RecipeInstaller{ic, discover, l, mv, f, e, v, ff, status, p, pi, lkf}
 
 	err := i.Install()
 	require.Error(t, err)
 }
 
-func TestInstall_FailsOnNoOs(t *testing.T) {
```

Comment:
Do we not want to fail if the manifest contains an empty OS anymore?

Reply:
Yes we do want to fail, this test has been moved to os_validator_test.go